metric_wavefront.go

```diff
@@ -11,7 +13,13 @@ import (
 var wavefrontConfig *wavefront.WavefrontConfig
 
 func initMetricWavefront() error {
-	addr, err := net.ResolveTCPAddr("tcp", config.Metric.WavefrontUrl)
+	url := extractWavefrontUrl(config.Metric.WavefrontUrl)
```

Comment:
nit: `WavefontURL`, Url is not a word ;p

Reply:
Indeed. Thank you.